templates/docker-compose/docker-compose.prod.yml

```diff
@@ -53,12 +53,14 @@ services:
             - db
             - redis
             - mailer
+            - meilisearch
         environment:
             REDIS_HOST: ${CACHE_SERVICE_NAME}
             REDIS_PORT: ${REDIS_PORT}
             MAILER_SERVICE_NAME: ${MAILER_SERVICE_NAME}
             WS_SERVICE_NAME: ${WS_SERVICE_NAME}
             MQ_SERVICE_NAME: ${MQ_SERVICE_NAME}
+            MEILISEARCH_HOST: "http://${MEILISEARCH_SERVICE_NAME}:${MEILISEARCH_PORT}"
```

Comment:
Does it not make sense to put the value in `.env` given it's reused in `docker-compose.yml` several times?

Reply:
I've updated it